x-pack/plugins/reporting/server/export_types/printable_pdf/create_job/index.ts

```diff
@@ -7,26 +7,20 @@
 
 import { CreateJobFn, CreateJobFnFactory } from '../../../types';
 import { validateUrls } from '../../common';
-import { JobParamsPDF, JobParamsPDFLegacy, TaskPayloadPDF } from '../types';
-import { compatibilityShim } from './compatibility_shim';
+import { JobParamsPDF, TaskPayloadPDF } from '../types';
 
-/*
- * Incoming job params can be `JobParamsPDF` or `JobParamsPDFLegacy` depending
- * on the version that the POST URL was copied from.
- */
-export const createJobFnFactory: CreateJobFnFactory<
-  CreateJobFn<JobParamsPDF | JobParamsPDFLegacy, TaskPayloadPDF>
-> = function createJobFactoryFn(_reporting, logger) {
-  return compatibilityShim(async function createJobFn(
-    { relativeUrls, ...jobParams }: JobParamsPDF // relativeUrls does not belong in the payload of PDFV1
-  ) {
-    validateUrls(relativeUrls);
+export const createJobFnFactory: CreateJobFnFactory<CreateJobFn<JobParamsPDF, TaskPayloadPDF>> =
+  function createJobFactoryFn() {
+    return async function createJobFn(
+      { relativeUrls, ...jobParams }: JobParamsPDF // relativeUrls does not belong in the payload of PDFV1
+    ) {
+      validateUrls(relativeUrls);
 
-    // return the payload
-    return {
-      ...jobParams,
-      forceNow: new Date().toISOString(),
-      objects: relativeUrls.map((u) => ({ relativeUrl: u })),
+      // return the payload
+      return {
+        ...jobParams,
```

Comment:
I noticed this should have `isDeprecated: true` since we have a V2 type for this. I'm leaving that as a separate issue: https://github.com/elastic/kibana/issues/121365